src/components/style/index.css

```diff
@@ -39,3 +38,8 @@
 @import "./toaster.css";
 @import "./dialog.css";
 @import "./menu.css";
+/* Reach framework styles */
+@import "@reach/menu-button/styles.css";
+/* Tippy framework styles */
+@import "tippy.js/dist/tippy.css";
+@import "./tooltip.css";
```

Comment:
I think we have two options for organizing tippy themes:

1. Dedicated theme dir: `src/components/style/tippy/<theme_name>.css`
2. Inlined in component css: `src/components/style/<name_of_component_that_uses_tippy>.css`
 
I like option 1 because it's clear that the CSS is targeting tippy-specific selectors and solidifies the relationship of that CSS to the third party theming conventions.

I like option 2 because if a developer thinks "i need to change tooltip styles", they don't have to do extra digging to find the tippy theme.

Interested in your thoughts here.

Reply:
Hmmmm yeah this is damn if you do damn if you don't. If we choose option 1, then people who go to edit the "tooltip css" are going to be confused. If we choose option 2, there's a liability that someone can create an identical theme which can mess with another.

The big issue here is that we're not just applying a theme, we're applying a "type".

I'm trying to think if there is a "combination" route we can go here. like we have a place where we put the themes, but then we import the themes into a specific component. That way you get all the separated themes in one place, you still have the component specific css, which is helpful for people to find their way.

Reply:
I like the idea of creating a top-level dir for tippy themes `components/style/tippy/<name>-theme.css` and importing them directly in component CSS files instead of into the index CSS

Reply:
do you mind if I set up that structure in the v10 popover ticket?